bundle-workflow/python/test_workflow/test_cluster.py

```diff
@@ -3,8 +3,26 @@
 import urllib.request
 import shutil
 import subprocess
+import abc
 
-class LocalTestCluster:
+class TestCluster(abc.ABC):
```

Comment:
Can we pull the interface in a separate file? The other clusters like PerfTest, IntegTest can use the same

Reply:
Sure